modules/Bio/EnsEMBL/Variation/Pipeline/ReleaseDataDumps/FinishDumpPopulation.pm

```diff
@@ -32,14 +32,15 @@ package Bio::EnsEMBL::Variation::Pipeline::ReleaseDataDumps::FinishDumpPopulatio
 use base ('Bio::EnsEMBL::Variation::Pipeline::ReleaseDataDumps::BaseDataDumpsProcess');
 
 use FileHandle;
+use Compress::Zlib;
 
 sub run {
     my $self = shift;
     my $file_type = $self->param('file_type');
     if ($file_type eq 'gvf') {
         # remove allele_string and variation_id attributes from file    
         # gzip file
-        $self->clean_up_gvf_file;
+#        $self->clean_up_gvf_file;
```

Comment:
Is this meant to be hashed out?

Reply:
It was taking too long and I run the cleanup separately. It has now been added as a semaphore to the pipeline.